tests/apps/loqus/conftest.py

```diff
@@ -54,90 +77,147 @@ def loqus_config():
 
 
-@pytest.fixture(scope='function')
+@pytest.fixture(scope="function")
+def loqusdbapi_exception(loqus_config, loqus_process_exception):
+    """
+        loqusdb API fixture
+    """
+
+    _loqus_api = LoqusdbAPI(loqus_config)
+    _loqus_api.process = loqus_process_exception
+
+    return _loqus_api
+
+
+@pytest.fixture(scope="function")
 def loqusdb_output():
     """
         loqusdb stderr for a successful load
     """
     return LOQUSDB_OUTPUT
 
 
-@pytest.fixture(scope='function')
+@pytest.fixture(scope="function")
 def loqusdb_case_output():
     """
         loqusdb output for a 'loqusdb cases -c <case_id> --to-json' command
     """
 
-    _output = (b'[{"_id": "1234", "case_id": "a_case", '
-               b'"vcf_path": "test.vcf.gz", "vcf_sv_path": null, "nr_variants": 15, '
-               b'"nr_sv_variants": null, "profile_path": "test.vcf.gz", "individuals": '
-               b'[{"ind_id": "proband", "name": "proband", "case_id": "recessive_trio", '
-               b'"ind_index": 2, "sex": 1, "profile": ["CC", "TT", "TT", "CC", "TT", '
-               b'"AA", "CC", "GG", "GG", "GG", "TT", "GG", "CC", "AA", "GG", "GG", "GG", '
-               b'"AA", "CC", "CC", "CC", "GG", "CC", "CC", "TT", "CC", "GG", "GG", "TT", '
-               b'"AA", "AA", "GG", "CC", "AA", "AA", "AA", "AA", "GG", "GG", "TT", "GG", '
-               b'"AA", "TT", "AA", "GG", "GG", "CC", "AA", "TT", "TT"]}, {"ind_id": '
-               b'"mother", "name": "mother", "case_id": "recessive_trio", "ind_index": 1, '
-               b'"sex": 2, "profile": ["CC", "TT", "TT", "CC", "TT", "AA", "CC", "GG", '
-               b'"GG", "GG", "TT", "GG", "CC", "AA", "GG", "GG", "GG", "AA", "CC", "CC", '
-               b'"CC", "GG", "CC", "CC", "TT", "CC", "GG", "GG", "TT", "AA", "AA", "GG", '
-               b'"CC", "AA", "AA", "AA", "AA", "GG", "GG", "TT", "GG", "AA", "TT", "AA", '
-               b'"GG", "GG", "CC", "AA", "TT", "TT"]}, {"ind_id": "father", "name": '
-               b'"father", "case_id": "recessive_trio", "ind_index": 0, "sex": 1, '
-               b'"profile": ["CC", "TT", "TT", "CC", "TT", "AA", "CC", "GG", "GG", '
-               b'"GG", "TT", "GG", "CC", "AA", "GG", "GG", "GG", "AA", "CC", "CC", '
-               b'"CC", "GG", "CC", "CC", "TT", "CC", "GG", "GG", "TT", "AA", "AA", '
-               b'"GG", "CC", "AA", "AA", "AA", "AA", "GG", "GG", "TT", "GG", "AA", '
-               b'"TT", "AA", "GG", "GG", "CC", "AA", "TT", "TT"]}], "sv_individuals": '
-               b'[], "_inds": {"proband": {"ind_id": "proband", "name": "proband", '
-               b'"case_id": "recessive_trio", "ind_index": 2, "sex": 1}, "mother": '
-               b'{"ind_id": "mother", "name": "mother", "case_id": "recessive_trio", '
-               b'"ind_index": 1, "sex": 2}, "father": {"ind_id": "father", "name": '
-               b'"father", "case_id": "recessive_trio", "ind_index": 0, "sex": 1}}, '
-               b'"_sv_inds": {}}]\n')
+    _output = (
```

Comment:
I think you could use the package snapshot for this purpose

Reply:
Ok we discussed this irl